### DISCUSSION TITLE
Help Build Inav for Windows Light didn't work

### DISCUSSION THREAD

**Discussion by abimaputra78** [iNavFlight/inav#7414] [Q&A]
Dear Inav,

I need your Help,
I had followed your instruction in https://github.com/iNavFlight/inav/blob/master/docs/development/Building%20in%20Windows%20light.md for building Inav in Window Light. I had downloaded and installed the following
 
Git
ARM Toolchain
Ruby 
Toolset Scripts from DropBox

but when I opened console, and type this 
=> arm-none-eabi-gcc --version 
I always had this issue as per attached console picture. 
Would you please help and give solution for the below issue since I used Windows 7. I have no Ideas of this honestly. Thanks so much.

Rgds
Dit.

![image](https://user-images.githubusercontent.com/49124286/131530954-5676fe1b-0713-437e-a063-f1db2d3c6ffa.png)

**Comment by DzikuVx** [comment]
We no longer support building with Win7. You can either migrate to windows 10 WSL or use Linux
